.rubocop.yml

```diff
@@ -3,10 +3,11 @@ AllCops:
   Include:
     - "**/Gemfile"
     - "**/lib/**.rb"
+    - "**/lib/**/**.rb"
```

Comment:
Is there a reason we can't use `"**/lib/**/*"` to replace line 5 & 6? I also wonder if in the long run it might be easier to maintain an `Exclude` list, rather than `Include`.